packages/dds/tree/src/test/simple-tree/objectNode.spec.ts

```diff
@@ -409,7 +409,7 @@ describeHydration(
 			const allowed = [Note] as const;
 			{
 				type X = InsertableTreeNodeFromAllowedTypes<typeof allowed>;
-				const test: X = [{}];
+				const test: X = {};
```

Comment:
I'm not sure I understand this change. For my education, if nothing else, what prompted this?

Reply:
The old code was incorrect, and stopped compiling due to the fix, since an array of objects has keys (ex: length) and thus can't be used to construct an empty object node.

Reply:
Ah, okay. I thought maybe the previous syntax had been a workaround or something, but didn't understand how it worked. So just bugs that were caught and have now been fixed. Great!